_posts/2017-05-02-newsletter.md

```diff
@@ -0,0 +1,222 @@
+- Built in LSP support (**@tjdevries**, https://github.com/neovim/neovim/pull/6856) (see http://langserver.org/ why that could be of interest to you)
+- Cursorline and cursorcolumn respecting syntax highlighting (**@zhou13**, https://github.com/neovim/neovim/pull/6380)
+
+
+Around Neovim
+------------------
+
+The wiki page of [related
+projects](https://github.com/neovim/neovim/wiki/Related-projects) has seen quite
+some additions, check out the full changes [here](https://github.com/neovim/neovim/wiki/Related-projects/_compare/7ce8559...e2c051fd85226ff6a67c67f9de55ad9039aa2948).
+
+
+### API clients
+
+API clients are at the heart of Neovim's architecture, and significantly improve developer experience. Write a plugin in
+any language you want! Since the last newsletter, we've seen the addition of another [C++](https://github.com/DaikiMaekawa/neovim.cpp) client, one for [Elixir](https://github.com/dm1try/nvim), and a [Racket](https://gitlab.com/HiPhish/neovim.rkt) client. The [node client](https://github.com/neovim/node-client) got a an overhaul and a new maintainer. Thanks **@billyvg** for taking over! While not totally new, a special shoutout goes to the [ruby client](https://github.com/alexgenco/neovim-ruby) for being very well-maintained. Thanks, **@alexgenco**!
+
+
+If you happen to prefer a language not yet listed, a good starting point are the [docs](https://neovim.io/doc/user/msgpack_rpc.html#RPC), and be sure to come talk about it on the [gitter channel](gitter.im/neovim).
+
+### GUIs
+
+Too much activity has happened on the GUI front to list all new clients, so let's just have a look at some that stand out. VSCode  is using Neovim to properly [integrate ex-mode commands](https://github.com/VSCodeVim/Vim), and Sublime Text 3 gained a [neovim input mode](https://github.com/lunixbochs/actualvim). The latter also makes use of another
+cool neovim specific feature, the externalized popumenu, and it only took them
+[50 LOC](https://github.com/lunixbochs/ActualVim/issues/57#issuecomment-286452725). Last but not least, [eovim](https://github.com/jeanguyomarch/eovim/) is an [enlightenment](https://www.enlightenment.org/) client for Neovim.
+
+### The Architecture of Neovim.
+
+Some students have studied the Architecture of Neovim and published an
+"analytical essay" about their findings. Be sure to
+[have a look](https://delftswa.gitbooks.io/desosa-2017/content/neovim/chapter.html)
+if you are interested in the bigger picture.
+
+### Vimcasts and Modern Vim
+
+Drew Neil of [vimcasts](http://vimcasts.org) fame has recorded [10 casts](http://vimcasts.org/blog/2017/06/dive-into-neovim/) about Neovim, and began work on a successor to his praised book [Practical Vim](https://pragprog.com/book/dnvim2/practical-vim-second-edition) called [Modern Vim](http://vimcasts.org/blog/2017/05/working-title-modern-vim/). Most of the book will be  suitable for Vim 8 and Neovim users, but about 1/3rd of the book will cover Neovim-specific functionality.
```

Comment:
I'd suggest changing a couple of the links:

* instead of http://vimcasts.org/blog/2017/06/dive-into-neovim/ you could link to the [Neovim category](http://vimcasts.org/categories/neovim/). As I publish more screencasts and articles about Neovim they will appear on that page.
* instead of http://vimcasts.org/blog/2017/05/working-title-modern-vim/ you could link to the [Modern Vim page on Pragmatic Bookshelf](https://pragprog.com/book/modvim/modern-vim). Modern Vim has now been published as a beta book.

Reply:
Good news,thanks for bringing it up!